sig-scalability/processes/scalability-validation.md

```diff
@@ -123,6 +123,6 @@ Responsibilities lying with other SIGs/teams as applicable (could be sig-scalabi
 [supporting]: http://blog.kubernetes.io/2017/03/scalability-updates-in-kubernetes-1.6.html
 [here]: https://docs.google.com/document/d/15rD6XBtKyvXXifkRAsAVFBqEGApQxDRWM3H1bZSBsKQ
 [#47865]: https://github.com/kubernetes/kubernetes/issues/47865
-[test_owners.csv]: https://github.com/kubernetes/kubernetes/blob/master/test/test_owners.csv
+[test_owners.csv]: https://github.com/kubernetes/kubernetes/blob/master/test/OWNERS
```

Comment:
`test_owners.csv` was removed in https://github.com/kubernetes/kubernetes/pull/84070
I don't think there's a direct replacement. It might be better to remove the link and update the wording in the _[Responsibilities lying with other SIGs/teams as applicable (could be sig-scalability too):](https://github.com/kubernetes/community/blob/master/sig-scalability/processes/scalability-validation.md#clearly-lay-down-responsibilities-for-scale-tests)_ section

Reply:
This is top-level owners files, which isn't great. We don't want to assign all bugs to those people.

We should fix the line 115 to:
"Owner/SIG for the e2e test (as specified by corresponding OWNERS file)"

I think it would be better (even though it doesn't provide he exact link).

Reply:
ths, i 'll close the pr